src/ai/backend/manager/models/agent.py

```diff
@@ -197,7 +197,25 @@ async def resolve_hardware_metadata(
 
     async def resolve_local_config(self, info: graphene.ResolveInfo) -> Mapping[str, Any]:
         graph_ctx: GraphQueryContext = info.context
-        return await graph_ctx.registry.get_agent_local_config(self.id, self.addr)
+
+        def _convert_none_to_str(data: Mapping[str, Any]) -> Mapping[str, Any]:
+            converted: MutableMapping[str, Any] = {}
+            for k, v in data.items():
+                if v is None:
+                    converted[k] = ""
+                    continue
+                if isinstance(v, dict):
+                    converted[k] = _convert_none_to_str(v)
+                    continue
+                converted[k] = v
+            return converted
```

Comment:
* Let's refactor out this function as it is also duplicate in `src/ai/backend/manager/models/kernel.py`. Probably `ai.backend.common.utils` would be a good place to put this kind of small utility functions, though `common.utils` should be always considered as "to-be-refactored".
* I'd suggest a more concise naming: `coerce_none`.
* Since the function is coercing None to the empty string always, I believe it's assuming that `data` is `Mapping[str, str | None]`. Am I correct?
* Let's rewrite the body using `if-elif-else` blocks instead of inserting `continue` in every `if` branch.

Reply:
reflexed. but I am not sure whether the 3rd question is correct or not. cause I guess the Value of the map can be Mapping, None or anything etc